rst_files/kalman.rst

```diff
@@ -101,71 +101,42 @@ This density :math:`p(x)` is shown below as a contour map, with the center of th
 .. code-block:: julia
   :class: collapse
 
-  using Plots, LaTeXStrings, LinearAlgebra
+  using Plots, LaTeXStrings, LinearAlgebra, Compat, Distributions
     gr(fmt=:png)
```

Comment:
The indentation of this whole cell is off. Using 4 spaces.

Reply:
Thanks. I think it's hard to enforce indentation and the like cross-platform, so as long as it doesn't break the rST (that is, unless indentation _levels_ are off), I'd rather not worry about it.

Reply:
The levels are off. In the Jupyter notebook, every line except the first has a leading two spaces.